implementations/rust/ockam/ockam_app/src/shared_service/tcp_outlet/create.rs

```diff
@@ -6,15 +6,19 @@ use tracing::{debug, error, info};
 
 use crate::app::AppState;
 use crate::error::Error;
+use crate::invitations::build_args_for_create_service_invitation;
+use crate::invitations::commands::create_service_invitation;
 
 /// Create a TCP outlet within the default node.
 #[tauri::command]
 pub async fn tcp_outlet_create(
     app: AppHandle<Wry>,
     service: String,
     port: String,
+    email: String,
 ) -> Result<(), String> {
-    tcp_outlet_create_impl(app, service, port)
+    let email = if email.is_empty() { None } else { Some(email) };
```

Comment:
🤔 Given the desired use case, I think we eventually will want to be able to supply 0, 1, or several emails. Comma-separated or maybe newline-separated, for the form input, which might need to change to a textedit.

Reply:
Does the API endpoint support multiple recipient emails already?

Reply:
Today, not as a single request - that would have to be multiple backend requests for each, though they could fire in parallel. If we did this often a bulk endpoint would probably make sense.